Opcodes/partikkel.h

```diff
@@ -1,6 +1,6 @@
 /*
 Partikkel - a granular synthesis module for Csound 5
-Copyright (C) 2006-2009 �yvind Brandtsegg, Torgeir Strand Henriksen,
+Copyright (C) 2006-2009 �yvind Brandtsegg, Torgeir Strand Henriksen,
```

Comment:
Hmm why did this change?

Reply:
Hmm, something must have gone wrong with git I'll fix it

Reply:
Done